packages/components/src/drop-zone/provider.js

```diff
@@ -35,6 +35,12 @@ const getDragEventType = ( { dataTransfer } ) => {
 	return 'default';
 };
 
+const isTypeSupported = ( type, typesSupported ) => {
```

Comment:
the `typesSupported` name is misleading, it supposes an array while it's an object. I think `dropzone` (because it's the dropzone object is more correct).

Reply:
Oh, I see what you mean. Changed method to `isTypeSupportedByDropZone` and variable name to `dropZone`.